components/AvalancheProblemImage.tsx

```diff
@@ -50,7 +44,7 @@ export const AvalancheProblemImage: React.FunctionComponent<AvalancheProblemImag
       {!gallery ? (
         <View style={styles.container}>
           <Image source={{uri: media.url.original}} style={{width: '70%', aspectRatio: imageDimensions.height / imageDimensions.width}} />
-          <RenderHTML source={{html: media.caption}} baseStyle={baseStyle} contentWidth={width} />
+          <HTMLRenderer source={{html: `<em>${media.caption}</em>`}} />
```

Comment:
Is it always safe to add HTML tags to this? What assumptions must be true about `media.caption` for that to be valid?

Reply:
I think the assumptions are it either has to be a naked string (in which case it's safe) or a valid HTML fragment (in which case it's safe). 

If it's an invalid HTML fragment (unclosed tags or something) then it's gonna break - but it would break without it too.

Is there another case I'm missing that you can think of?

Reply:
I was wondering what happens if the HTML starts with like doctype and head .... and are there any other edges? If you think no, then we're good to go!

Reply:
none of the APIs seem to be returning full pages currently - they just return fragments AFAICT. not saying that someone couldn't break that in the future 😬

Reply:
Yeah it's resiliency in the face of whatever nonsense we get from upstream I am more worried about ...